Consistence/Sniffs/Exceptions/ExceptionDeclarationSniff.php

```diff
@@ -50,13 +54,39 @@ public function process(PhpCsFile $phpcsFile, $classPointer)
 			), $classPointer, self::CODE_NOT_ENDING_WITH_EXCEPTION);
 		}
 
+		$this->checkExceptionDirectoryName($phpcsFile, $classPointer);
+
 		$this->checkThatExceptionIsChainable($phpcsFile, $classPointer);
 	}
 
 	/**
 	 * @param \PHP_CodeSniffer\Files\File $phpcsFile
 	 * @param int $classPointer
 	 */
+	private function checkExceptionDirectoryName(PhpCsFile $phpcsFile, int $classPointer)
+	{
+		$filename = $phpcsFile->getFilename();
+		$filename = str_replace('\\', '/', $filename); // normalize path for Windows
```

Comment:
Can you please test on Windows if this is really needed? I have the feeling that PHPCS was already converting all the paths to linux style (but I am not sure if at this exact place).

Will have to set up Appveyor at some point, thought its really unrealiable :(

Reply:
It is really needed, because PHPCS detects the paths with backslashes.